TrafficSimulator/UserSettings/EditorUserSettings.asset

```diff
@@ -6,7 +6,7 @@ EditorUserSettings:
   serializedVersion: 4
```

Comment:
This should not be readded to the project. `UserSettings` is in the gitignore, but you haven't cleared your tracked files. Do that so you don't introduce ignored files to the repository